webui/react/src/e2e/models/pages/Admin/UserManagement.ts

```diff
@@ -79,7 +79,13 @@ export class UserManagement extends AdminPage {
     });
     expect(
       filteredRows,
-      `name:${name}, component:${await this.table.table.rows.pwLocator.allInnerTexts()}`,
+      `name: ${name}
+      users: ${(await this.table.table.rows.user.pwLocator.allInnerTexts()).map((item) =>
+        item.replace(/(\r\n|\n|\r)/gm, ''),
```

Comment:
nitpick: can we pull this map+regex out into it's own constant/function with a descriptive name? Or is there a built in function for `trim`ing whitespace?